iati/core/default.py

```diff
@@ -8,7 +8,10 @@
     Implement more than Codelists.
 
 """
+
+from collections import defaultdict
```

Comment:
Imports with a `from` keyword should be placed after those with a straight `import` statement:
- https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting
- https://stackoverflow.com/a/20763446/2761030

Reply:
More specific reference: https://github.com/reddit/reddit/wiki/PythonImportGuidelines

Reply:
https://github.com/IATI/iati.core/pull/142/commits/3cf58da9c1d903eb3ddc4cc46a679ab4a3fef0dd